src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2095,6 +2097,15 @@ export class ResultsViewPageStore {
         return parsedQString.query.session_id;
     }
 
+    private getCytoband(hugoGeneSymbol:string):string {
+        const referenceGenes = this.referenceGenes!.result;
+        if (referenceGenes) {
+            const gene = _.find(referenceGenes, (g: ReferenceGenomeGene) => g.hugoGeneSymbol === hugoGeneSymbol);
+            return gene ? gene.cytoband : "";
```

Comment:
should we really use empty string here?  seems it would be better to use null until actual use